src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -90,41 +221,54 @@
           "type": "string"
         }
       },
+      "required": [
+        "scheduler"
+      ],
       "type": "object"
     },
     "preprocessing": {
+      "additionalProperties": false,
       "properties": {
         "lateral_boundary_conditions": {
+          "additionalProperties": false,
```

Comment:
When we get to the point of having a driver for creating the lateral boundary conditions instead of using staged files, we'll likely want to have more information in this section.